### DISCUSSION TITLE
Request coding for new device - Smart Water Metering

### DISCUSSION THREAD

**Discussion by tzeryin1981** [Koenkk/zigbee2mqtt#15735] [Devices]
![Screenshot_20221224_081719_Facebook](https://user-images.githubusercontent.com/96242882/209432789-d29ef296-5c0c-43a9-8f30-de48990c66bb.jpg)

Model: TS0601
Manufacturer: _TZE200_vuwtqx0t
Hope there is a coding for this water metering and valve function

**Comment by Perfect-Web** [comment]
any news here ? i would like support for this as well

**Comment by michaeleverist** [comment]
Ive made slight progress here: [https://github.com/Koenkk/zigbee2mqtt/issues/21255](https://github.com/Koenkk/zigbee2mqtt/issues/21255)  but still need help or a better understanding on how it all works.

Total consumption and value open/close work as well as a few other things.  Most of the other stuff is there but has issues with the converters etc being used.
